force-app/main/default/objects/Thread__c/fields/HOT_DelPol_ToDeleteRecord__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>HOT_DelPol_ToDeleteRecord__c</fullName>
+    <externalId>false</externalId>
+    <formula>IF(AND(ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BRUKER-FORMIDLER&apos;), ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BRUKER-TOLK&apos;), ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BESTILLER-FORMIDLER&apos;), (TODAY() - DATEVALUE(CreatedDate)) &gt; 365), true, false)</formula>
```

Comment:
Nå vil den vel aldri være `true` da den aldri vil ha ulike verdier i `CRM_Thread_Type__c`? Må vel ha en OR på verdiene i `CRM_Thread_Type__c`, og så en AND på alder.